examples/notebooks/beam-ml/run_inference_streaming_pipeline.ipynb

```diff
@@ -0,0 +1,292 @@
+      ],
+      "metadata": {
+        "id": "A0v-oGSfsTHh"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "metadata": {
+        "id": "hNslPJGil2Zc"
+      },
+      "execution_count": 63,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Import dependencies and set up your bucket\n",
+        "Use the following code to import dependencies and to set up your Google Cloud Storage bucket.\n",
+        "\n",
+        "Replace `MESSAGE_TOPIC` and `RESPONSE_TOPIC` with the Pub/Sub topics in your project.\n",
+        "\n",
+        "**Important**: If an error occurs, restart your runtime."
+      ],
+      "metadata": {
+        "id": "I7vMsFGW16bZ"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "import apache_beam as beam\n",
+        "from apache_beam.options.pipeline_options import PipelineOptions\n",
+        "from apache_beam.ml.inference.base import PredictionResult\n",
+        "from apache_beam.ml.inference.base import RunInference\n",
+        "from apache_beam.ml.inference.pytorch_inference import make_tensor_model_fn\n",
+        "from apache_beam.ml.inference.pytorch_inference import PytorchModelHandlerTensor\n",
+        "import torch\n",
+        "from transformers import AutoConfig\n",
+        "from transformers import AutoModelForSeq2SeqLM\n",
+        "from transformers import AutoTokenizer\n",
+        "from transformers.tokenization_utils import PreTrainedTokenizer\n",
+        "\n",
+        "message_topic = \"MESSAGE_TOPIC\"\n",
+        "response_topic = \"RESPONSE_TOPIC\"\n",
+        "\n",
+        "MAX_RESPONSE_TOKENS = 256\n",
+        "\n",
+        "model_name = \"google/flan-t5-small\"\n",
+        "state_dict_path = \"saved_model\""
+      ],
+      "metadata": {
+        "id": "uhbOYUzvbOSc"
+      },
+      "execution_count": 75,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Download and save the model\n",
+        "We will use the AutoClasses from Hugging Face to instantly load the model in memory and later save it to the path defined above."
+      ],
+      "metadata": {
+        "id": "yRls3LmxswrC"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "model = AutoModelForSeq2SeqLM.from_pretrained(\n",
+        "        model_name, torch_dtype=torch.bfloat16\n",
+        "    )\n",
+        "\n",
+        "directory = os.path.dirname(state_dict_path)\n",
+        "torch.save(model.state_dict(), state_dict_path)"
+      ],
+      "metadata": {
+        "id": "PKhkiQFJe44n"
+      },
+      "execution_count": 76,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Utitlity functions for before/after running RunInference"
+      ],
```

Comment:
Add a line explaining what the utility functions are for. Something like:

These utility functions are used before and after the RunInference transform to....